src/main/java/no/finn/unleash/Unleash.java

```diff
@@ -15,6 +16,14 @@ default boolean isEnabled(String toggleName, UnleashContext context, boolean def
         return isEnabled(toggleName, defaultSetting);
     }
 
+    default boolean isEnabled(final String toggleName, final BiFunction<String, UnleashContext, Boolean> fallbackAction) {
+        return isEnabled(toggleName, false, fallbackAction);
+    }
+
+    boolean isEnabled(final String toggleName, boolean defaultSetting, final BiFunction<String, UnleashContext, Boolean> fallbackAction);
```

Comment:
By not providing fallback for new interface methods we will break existing implementations of the interface. This is probably ok, but needs to be at least a minor.

Reply:
Added a default implementation like for the other methods